docker/Dockerfile

```diff
@@ -109,7 +109,7 @@ COPY hail_builds/ ./hail_builds
 COPY hail_scripts/ ./hail_scripts
 COPY luigi_pipeline/ ./luigi_pipeline
 
-RUN python3 -m pip install -r ./luigi_pipeline/requirements.txt
+RUN python3 -m pip install -r ./requirements.txt
```

Comment:
I would see if you can try copying in the requirements.txt file alone and running this step as early on in the dockerfile as our installation process allows. Assuming the requirements.txt changes less often than the hail_* and luigi_* directories, we'll have to run this pip install any time something in those folders changes.

If we copy in the requirements.txt into the image individually earlier in the dockerfile, we can take advantage of more cache hits in the docker build, and only run that pip install step when the requirements.txt file changes.

Reply:
👍

Reply:
I lifted up the `pip install` to be before copying files from the repo into the docker build.  The dockerfile can probably use quite a bit more tlc but it's probably out of scope for this pr.

Reply:
Yeah, sounds good; only happened to notice this line since it appeared in the diff here 👍